selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -153,6 +157,9 @@ def publish(self, sm, pm):
     longitudinalPlan.modelMonoTime = sm.logMonoTime['modelV2']
     longitudinalPlan.processingDelay = (plan_send.logMonoTime / 1e9) - sm.logMonoTime['modelV2']
 
+    longitudinalPlan.xs = self.x_desired_trajectory.tolist()
+    longitudinalPlan.zs = self.z_model.tolist()
+
```

Comment:
Want to call these `distances` and `heights` to be consistent with the other names?